packages/netlify/test/functions/fixtures/middleware/astro.config.mjs

```diff
@@ -5,6 +5,7 @@ export default defineConfig({
   output: 'server',
   adapter: netlify({
     edgeMiddleware: process.env.EDGE_MIDDLEWARE === 'true',
+    imageCDN: process.env.DISABLE_IMAGE_CDN ? false : undefined,
```

Comment:
Should this be `=== "true"`?

Reply:
No, because then it would be `true` (aka: explicitly enabled) in cases where the env var isn't set. I want to test the default case `undefined`, where nothing was specified.